extras/nginx-mtls-auth/README.md

```diff
@@ -0,0 +1,39 @@
+# Sample of using securing Model Server with nginx
+
+Dockerfile and scripts in this directory are an example of using Nginx mTLS module to implement authentication and authorization of OpenVINO Model Server.
+
+This can secure both GRPC and REST endpoints. Model Server will present server certificate, and allow connection only from clients who perform full TLS handshake (successful client certificate authentication).
+
```

Comment:
Add a reference to external docs about MTLS and include a picture with the OVMS/NGINX architecture.

Reply:
added link to rfc